docs/data/scheduler/overview/overview.md

```diff
@@ -0,0 +1,18 @@
+---
+productId: x-scheduler
+title: Scheduler- Overview
+packageName: '@mui/x-scheduler'
+githubLabel: 'component: scheduler'
```

Comment:
Technically, we need these only when this component is present. 🤷 
https://github.com/mui/mui-x/blob/220a2de855070d384bede6c449f091fda3825767/docs/data/charts/overview/overview.md?plain=1#L12

Reply:
I don't have a strong preference here. 🤔 
I would probably lean towards adding the component and removing it in the future if we don't see a need for it. 🤷

Reply:
I'll add it then :+1: